internal/cmp/data_source_lb_pool_member_group.go

```diff
@@ -34,7 +34,15 @@ func (n *poolMemberGroupds) Read(ctx context.Context, d *utils.Data, meta interf
 	if err := d.Error(); err != nil {
 		return err
 	}
-
+	cmpVersion, err := utils.GetCmpVersion(n.rClient.Client)
+	if err != nil {
+		return err
+	}
```

Comment:
why do we need this logic here ? can you not make this as a function named getNSXTypefromCMP